models/release.go

```diff
@@ -197,16 +198,17 @@ func (opts *FindReleasesOptions) toConds(repoID int64) builder.Cond {
 }
 
 // GetReleasesByRepoID returns a list of releases of repository.
-func GetReleasesByRepoID(repoID int64, opts FindReleasesOptions, page, pageSize int) (rels []*Release, err error) {
-	if page <= 0 {
-		page = 1
+func GetReleasesByRepoID(repoID int64, opts FindReleasesOptions) (rels []*Release, err error) {
+	sess := x.
+		Desc("created_unix", "id").
+		Where(opts.toConds(repoID))
+
+	if opts.PageSize == 0 {
```

Comment:
why do you differ from https://github.com/go-gitea/gitea/pull/9452/files#diff-212b9fa4c927665ecbc5a5a36382ecedR69-R71?